src/main/scala/encry/api/http/routes/PeersApiRoute.scala

```diff
@@ -40,6 +46,24 @@ case class PeersApiRoute(override val settings: RESTApiSettings,
     (r => complete(r))
   )
 
+  def connectPeer: Route = path("connect") {
+    post(entity(as[String]) {
+      str =>
+        complete {
+          Try {
+            val split = str.split(':')
+            (split(0), split(1).toInt)
+          } match {
+              case Success((host, port)) =>
+                nodeViewSynchronizer ! PeerFromCli(new InetSocketAddress(host, port))
```

Comment:
We are trying to remove all direct imports from `EncryApp`, so, better pass ref to NvS as a constructor argument